lang/pt_br.json

```diff
@@ -121,7 +121,7 @@
   "GURPS.addHardenedDRLevel" : "RD Ampliada Nível",
   "GURPS.addInjuryTolerance" : "Tolerância a Danos",
   "GURPS.addInjuryToleranceUnliving" : "Não vivo",
-  "GURPS.addInjuryToleranceHomogeneous" : "Homogêneo",
+  "GURPS.addInjuryToleranceHomogenous" : "Homogêneo",
```

Comment:
Thanks for correcting the typo!

Reply:
No problem. Both are English words. 'Homogenous' and 'homogeneous' pretty much mean the same thing, but GURPS uses the former, not the latter, in the rules.